ci/pipeline_bm_stemcell.yml

```diff
@@ -0,0 +1,180 @@
+---
+jobs:
+- name: new-version
+  plan:
+  - get: published-stemcell
+  - get: bosh-softlayer-tools
+  - task: make-version
+    file: bosh-softlayer-tools/ci/tasks/make-stemcell-version.yml
+  - put: stemcell-version
+    params: {file: version/semver}
+
+- name: build-ubuntu-trusty-bm
+  plan:
+  - aggregate:
+    - {get: bosh-src}
+    - {get: bosh-softlayer-tools}
+    - {trigger: true, passed: [new-version], get: stemcell-version}
+  - task: build
+    file: bosh-softlayer-tools/ci/tasks/build-stemcell-baremetal.yml
+    params:
+      BOSH_AWS_ACCESS_KEY_ID:     {{aws_builder_access_key}}
+      BOSH_AWS_SECRET_ACCESS_KEY: {{aws_builder_secret_access_key}}
+      BOSH_AWS_SECURITY_GROUP:    {{aws_builder_sec_group}}
+      BOSH_KEYPAIR_NAME:          {{aws_builder_keypair_name}}
+      BOSH_PRIVATE_KEY:           {{aws_builder_private_key}}
+      IAAS: softlayer
+      HYPERVISOR: esxi
+      OS_NAME: ubuntu
+      OS_VERSION: trusty
+  - put: stemcell-ubuntu-trusty-raw
+    params: {from: build/bosh-stemcell-.*-softlayer-.*-raw.tgz}
+
+- name: build-ubuntu-trusty-fake
+  plan:
+  - aggregate:
+    - {get: bosh-softlayer-tools}
+    - {trigger: true, passed: [new-version], get: stemcell-version}
+  - task: build
+    file: bosh-softlayer-tools/ci/tasks/build-stemcell-fake.yml
+    params:
+      IAAS: softlayer
+      HYPERVISOR: esxi
+      OS_NAME: ubuntu
+      OS_VERSION: trusty
+  - put: stemcell-ubuntu-trusty-raw
+    params: {from: build/bosh-stemcell-.*-softlayer-.*-raw.tgz}
+
+- name: generate-bm-stemcell-fsa
+  plan:
+  - get: bosh-softlayer-tools
+  - get: stemcell-version
+    trigger: true
+    passed: [build-ubuntu-trusty-bm]
+  - get: stemcell-ubuntu-trusty-raw
+    passed: [build-ubuntu-trusty-bm]
+  - task: generate-bm-stemcell-fsa
+    file: bosh-softlayer-tools/ci/tasks/generate-bm-stemcell-fsa.yml
+    params:
+      bm_stemcell_private_key:  {{bm_stemcell_private_key}}
+      bm_root_password: {{bm_root_password}}
+      bm_sjc01: 10.112.249.91
```

Comment:
It's the bare metal machine we used to make the .fsa file, follow the [instruction](https://www.pivotaltracker.com/story/show/108729546) Jimmy gave us.